packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -79,64 +67,116 @@ export type TooltipProps = MergeRight<
 export const Tooltip = forwardRef<HTMLDivElement, TooltipProps>(
   function Tooltip(
     {
+      id,
       children,
       content,
       placement = 'top',
       delay = 150,
-      open: userOpen,
-      defaultOpen = false,
-      portal = false,
+      open,
       className,
-      style,
       ...rest
     },
     ref,
   ) {
-    const [isOpen, setIsOpen] = useState(defaultOpen);
-    const internalOpen = userOpen ?? isOpen;
-
-    const Container = portal ? FloatingPortal : Fragment;
-
-    const { refs, floatingStyles, context } = useFloating({
-      open: internalOpen,
-      onOpenChange: setIsOpen,
-      placement,
-      whileElementsMounted: autoUpdate,
-      middleware: [
-        offset((data) => {
-          // get pseudo element arrow size
-          const styles = getComputedStyle(data.elements.floating, '::before');
-          return parseFloat(styles.height);
-        }),
-        flip({
-          fallbackAxisSideDirection: 'start',
-        }),
-        shift(),
-        arrowPseudoElement,
-      ],
-    });
-
-    const { styles: animationStyles } = useTransitionStyles(context, {
-      initial: {
-        opacity: 0,
-      },
-    });
-
-    const { getReferenceProps, getFloatingProps } = useInteractions([
-      // Event listeners to change the open state
-      useHover(context, { move: false, delay }),
-      useFocus(context),
-      useDismiss(context),
-      useRole(context, { role: 'tooltip' }),
-    ]);
-
-    const mergedRef = useMergeRefs([ref, refs.setFloating]);
-
-    const childMergedRef = useMergeRefs([
-      (children as ReactElement & RefAttributes<HTMLElement>)
-        .ref as MutableRefObject<HTMLElement>,
-      refs.setReference,
-    ]);
+    const randomTooltipId = useId();
+
+    const [internalOpen, setInternalOpen] = useState(false);
+
+    const triggerRef = useRef<HTMLElement>(null);
+    const tooltipRef = useRef<HTMLDivElement>(null);
+    const mergedRefs = useMergeRefs([tooltipRef, ref]);
+
+    const controlledOpen = open ?? internalOpen;
+
+    // NOTE: This code is purely to add React controlled component ability to Popover API
+    useEffect(() => {
+      if (!tooltipRef.current || !triggerRef.current) return;
+
+      const trigger = triggerRef.current;
+      const tooltip = tooltipRef.current;
+
+      const handleMouseover = () => {
+        setInternalOpen(true);
+      };
+
+      const handleMouseout = (event: MouseEvent) => {
+        /* if we move mosue to tooltip, don't close */
+        const el = event.relatedTarget as Element | null;
+        if (el === tooltip) return;
```

Comment:
```suggestion
        /* if we move mouse to tooltip, don't close */
        const el = event.relatedTarget as Element | null;
        if (el === tooltip) return;
```
Og, trenger vi her å sjekke både trigger og tooltip?

Reply:
Kan du utdjupe? Når du har `mouseout` frå `trigger`, så trenger me ikkje sjekke om musa er der 🤔

Reply:
ah, right så så lenge Tooltip og Trigger har 0px mellom seg funger dette - kanskje vi skulle hatt en liten `setTimeout` debounce slik at den tåler f.eks 5px mellom tooltip og trigger? ☺️

Reply:
Ja eg han tenkt det samme, men eg greier å flytte musepeikaren min inn i tooltip 🤔 
Dette kan vel variere frå system til system, så ein timeout på 150ms er kanskje greit uansett

Reply:
We decided here to always close tooltip when mouse or focus leaves the trigger